jest.config.js

```diff
@@ -141,7 +141,7 @@ module.exports = {
   // testLocationInResults: false,
 
   // The glob patterns Jest uses to detect test files
-  testMatch: ['**/__jest__/**/*.[jt]s?(x)', '**/?(*.)+(spec|jest).[tj]s?(x)'],
+  testMatch: ['**/__jest__/**/*.js?(x)', '**/?(*.)+(spec|jest).js?(x)'],
```

Comment:
Indeed, `.ts` files can not be loaded into `jest` as is. Actually, this RegExp is wrong. For one it should be `js` rather than `js?`, and the `(x)` is not needed ;)

```suggestion
  testMatch: ['**/__jest__/**/*.js', '**/?(*.)+(spec|jest).js'],
```